packages/design-system/src/helpers/tokens/tokensConfigs/fontSizes.json

```
@@ -0,0 +1,50 @@
+{
+  "xxs": {
+    "value": "10px",
+    "type": "typography"
```

```suggestion
    "type": "fontSizes"
```
Typography is not only font size but other properties of a font, e.g., weight, decoration, etc.